### PR TITLE
Update Chiado config format and keys

### PR DESCRIPTION
**Pull request by dapplion**
Chiado config has not been updated to the latest mainnet set of keys. Follow the same format as gnosis/config.yml but with Chiado values.

See diff between gnosis config and chiado config with this PR applied: https://www.diffchecker.com/qqrs55PU/

### CODE REVIEW

**Comment by filoozom** [comment]
There's a pretty extensive diff between the previous and new configuration files, including `CONFIG_NAME` which doesn't seem to make sense?

```diff
3,6c3
< ATTESTATION_PROPAGATION_SLOT_RANGE: 32
< ATTESTATION_SUBNET_COUNT: 64
< ATTESTATION_SUBNET_EXTRA_BITS: 0
< ATTESTATION_SUBNET_PREFIX_BITS: 6
---
> BASE_REWARD_FACTOR: 25
9a7
> BLS_WITHDRAWAL_PREFIX: 0x00
13c11
< CONFIG_NAME: 'gnosis'
---
> CONFIG_NAME: chiado
18a17,27
> DOMAIN_AGGREGATE_AND_PROOF: 0x06000000
> DOMAIN_BEACON_ATTESTER: 0x01000000
> DOMAIN_BEACON_PROPOSER: 0x00000000
> DOMAIN_CONTRIBUTION_AND_PROOF: 0x09000000
> DOMAIN_DEPOSIT: 0x03000000
> DOMAIN_RANDAO: 0x02000000
> DOMAIN_SELECTION_PROOF: 0x05000000
> DOMAIN_SYNC_COMMITTEE: 0x07000000
> DOMAIN_SYNC_COMMITTEE_SELECTION_PROOF: 0x08000000
> DOMAIN_VOLUNTARY_EXIT: 0x04000000
> EFFECTIVE_BALANCE_INCREMENT: 1000000000
20c29,32
< EPOCHS_PER_SUBNET_SUBSCRIPTION: 256
---
> EPOCHS_PER_ETH1_VOTING_PERIOD: 64
> EPOCHS_PER_HISTORICAL_VECTOR: 65536
> EPOCHS_PER_RANDOM_SUBNET_SUBSCRIPTION: 256
> EPOCHS_PER_SLASHINGS_VECTOR: 8192
24c36,40
< GOSSIP_MAX_SIZE: 10485760
---
> HISTORICAL_ROOTS_LIMIT: 16777216
> HYSTERESIS_DOWNWARD_MULTIPLIER: 1
> HYSTERESIS_QUOTIENT: 4
> HYSTERESIS_UPWARD_MULTIPLIER: 5
> INACTIVITY_PENALTY_QUOTIENT: 67108864
27,29c43,48
< MAXIMUM_GOSSIP_CLOCK_DISPARITY: 500
< MAX_CHUNK_SIZE: 10485760
< MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 2
---
> MAX_ATTESTATIONS: 128
> MAX_ATTESTER_SLASHINGS: 2
> MAX_COMMITTEES_PER_SLOT: 64
> MAX_DEPOSITS: 16
> MAX_EFFECTIVE_BALANCE: 32000000000
> MAX_PROPOSER_SLASHINGS: 16
31d49
< MAX_REQUEST_BLOCKS: 1024
33,34c51,55
< MESSAGE_DOMAIN_INVALID_SNAPPY: 0x00000000
< MESSAGE_DOMAIN_VALID_SNAPPY: 0x01000000
---
> MAX_SEED_LOOKAHEAD: 4
> MAX_VALIDATORS_PER_COMMITTEE: 2048
> MAX_VOLUNTARY_EXITS: 16
> MIN_ATTESTATION_INCLUSION_DELAY: 1
> MIN_DEPOSIT_AMOUNT: 1000000000
36c57
< MIN_EPOCHS_FOR_BLOCK_REQUESTS: 33024
---
> MIN_EPOCHS_TO_INACTIVITY_PENALTY: 4
39a61,62
> MIN_SEED_LOOKAHEAD: 1
> MIN_SLASHING_PENALTY_QUOTIENT: 128
41,46c64,67
< PRESET_BASE: 'gnosis'
< PROPOSER_SCORE_BOOST: 40
< REORG_HEAD_WEIGHT_THRESHOLD: 20
< REORG_MAX_EPOCHS_SINCE_FINALIZATION: 2
< REORG_PARENT_WEIGHT_THRESHOLD: 160
< RESP_TIMEOUT: 10
---
> PRESET_BASE: gnosis
> PROPORTIONAL_SLASHING_MULTIPLIER: 1
> PROPOSER_REWARD_QUOTIENT: 8
> RANDOM_SUBNETS_PER_VALIDATOR: 1
50,52c71,75
< SUBNETS_PER_NODE: 2
< TERMINAL_BLOCK_HASH: 0x0000000000000000000000000000000000000000000000000000000000000000
< TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: 18446744073709551615
---
> SHUFFLE_ROUND_COUNT: 90
> SLOTS_PER_EPOCH: 16
> SLOTS_PER_HISTORICAL_ROOT: 8192
> TARGET_AGGREGATORS_PER_COMMITTEE: 16
> TARGET_COMMITTEE_SIZE: 128
54c77,78
< TTFB_TIMEOUT: 5
---
> VALIDATOR_REGISTRY_LIMIT: 1099511627776
> WHISTLEBLOWER_REWARD_QUOTIENT: 512
```

**Comment by dapplion** [comment]
Opened https://github.com/gnosischain/configs/pull/29 to fix the CONFIG_NAME. The list of keys of the new config must equal the gnosis one, which follows Ethereum mainnet, and just make sure that it's values are correct.

**Comment by rolfyone** [comment]
I can confirm teku would accept this config, where the previous config had too many missing elements from mainnet and would require manual edits...

File: chiado/config.yaml [hunk 1]
```diff
@@ -152,9 +47,83 @@ CAPELLA_FORK_EPOCH: 244224 # Wed May 24 2023 13:12:00 GMT+0000
...
+
+# Networking
+# ---------------------------------------------------------------
+# `10 * 2**20` (= 10485760, 10 MiB)
+GOSSIP_MAX_SIZE: 10485760
+# `2**10` (= 1024)
+MAX_REQUEST_BLOCKS: 1024
+# `2**8` (= 256)
+EPOCHS_PER_SUBNET_SUBSCRIPTION: 256
+# `MIN_VALIDATOR_WITHDRAWABILITY_DELAY + CHURN_LIMIT_QUOTIENT // 2` (= 33024, ~5 months)
```
Comment (rolfyone) [inline chiado/config.yaml:107]: the comment math is basically from mainnet which will mean the timings will be completely wrong, is this acceptable? Potentially remove things like `~5 months` if not, or adjust...
I think this one is closer to 30 days... but didn't double check my working...

Comment (dapplion) [reply chiado/config.yaml:107]: Updated comments in https://github.com/gnosischain/configs/pull/31 thanks for noticing! Regarding the value itself, that ship sailed on genesis. We have had no reports of users unable to sync due to it yet

**Pull request merged**